src/Pokeview/components/Type.scss

```diff
@@ -0,0 +1,122 @@
+@mixin type-content {
+    height: 60%;
```

Comment:
I recommend you to use em instead of percentage units

Reply:
Fixed